data/changelog.txt

```diff
@@ -1,30 +1,90 @@
 # 0.H
 
 ## Features:
+Prevent occlusions via (auto)-semi-transparent sprites (similar to retraction)
+flip overmap tiles to default true
+Parametric basecamp expansion definitions
 
 
 ## Content:
+Add 14 scores, 24 achievements and 5 conducts related to vehicular travel
+Generic finales for generic mine
+New boss for physics lab
+4 New Apartment Complexes
+Add several new bungalows
 
 
 ## Interface:
```

Comment:
This is just because I didn't have a summary line for it, but think it could use mention
```suggestion
## Interface:
Show crafting failure chances in the crafting interface
```

Reply:
Huh, not sure why this one didn't get committed

```suggestion
## Interface:
Show crafting failure chances in the crafting interface
```